client/jetpack-cloud/sections/agency-dashboard/sites-overview/utils.ts

```diff
@@ -332,7 +332,7 @@ const formatBoostData = ( site: Site ) => {
 	const boostData: BoostNode = {
 		type: 'boost',
 		data: site.jetpack_boost_scores,
-		status: 'active',
+		status: 'inactive',
```

Comment:
Let's keep it `active` for now. We will have to refactor some code for types here, which we will do later.

Reply:
I kept it as `active` for now and can plan on fixing it during a refactor.